less/navigation.less

```diff
@@ -1 +1,41 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+header{
+        h1{
+            color: black;
+            .bordered();
+            padding: 2% 0 5px 35%;
+            @media @phone {
+                padding: 0% 5% 2% 25%;
+            }
+            
+        }
+        nav{
+      margin-top: 10px; 
+      padding: 0% 26% 1%;
+      justify-content: space-between;
+      display: flex;
+      @media @phone {
+        padding: 0 5% 2% 5%;
+
+      }
+
+
+      a{
```

Comment:
remember who the parent of this a tag is.  try to keep the children inside the parent